addons/l10n_it_edi/tests/export_xmls/bill_reverse_charge.xml

```diff
@@ -71,6 +71,7 @@
                     Product B, taxed 4%
                 </Descrizione>
                 <Quantita>1.00</Quantita>
+                <UnitaMisura>Pack of 6</UnitaMisura>
```

Comment:
I don't understand why we would be adding in unit of measures to an existing test? We shouldn't be changing any existing test flows :thinking:

Reply:
Same reason for this: https://github.com/odoo/odoo/pull/184131#discussion_r1897795956